build.sbt

```diff
@@ -176,45 +192,74 @@ lazy val `scalafix-input` = project
   )
   .dependsOn(`scalafix-data212`)
 
-val `scalafix-output` = MultiScalaProject("scalafix-output",
-                                          "scalafix/output",
-                                          _.settings(sharedScalafixSettings)
-                                            .settings(dontPublish)
-                                            .disablePlugins(ScalafixPlugin))
-
-lazy val output212    = Def.setting((baseDirectory in ThisBuild).value / "scalafix/output212/src/main")
-lazy val addOutput212 = unmanagedSourceDirectories in Compile += output212.value / "scala"
+val `scalafix-output` = MultiScalaProject(
+  "scalafix-output",
+  "scalafix/output",
+  _.settings(sharedScalafixSettings)
+    .settings(dontPublish)
+    .disablePlugins(ScalafixPlugin)
+)
 
-lazy val output212Plus =
-  Def.setting((baseDirectory in ThisBuild).value / "scalafix/output212+/src/main")
-lazy val addOutput212Plus = unmanagedSourceDirectories in Compile += output212Plus.value / "scala"
+val sources = Map( // scala211   scala212   scala213   scala213Pre
```

Comment:
Since .unsorted is only available in 2.13.0-pre-... I need a way to make the output compile with a complex build matrix. I tought it would make more sense to just remove the directory structure (output212, output212Plus, output213) and make this more fine-grained, per-file.